src/main.rs

```diff
@@ -90,35 +91,36 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
                                 .long("name")
                                 .multiple(true)
                                 .takes_value(true)
-                                .help("Chip Name"))
+                                .help("Chip Name")) 
```

Comment:
Extra space after the command. I recommend using to install "Trailing Space" plugin to Visual Studio Code which is able to capture this issues.

Reply:
Thank you, I allowed automatic trailing whitespace trimming.